plugins/search-elasticsearch-rest/src/main/java/com/thinkbiganalytics/search/ElasticSearchRestService.java

```diff
@@ -118,15 +116,12 @@ public void delete(@Nonnull String indexName, @Nonnull String typeName, @Nonnull
             log.error("Http protocol error for delete document for index={" + indexName + "}, type={" + typeName + "}, id={" + id + "}", clientProtocolException);
         } catch (IOException ioException) {
             log.error("IO Error in rest client", ioException);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "IO Error in rest client" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)